### ISSUE TITLE
loss is negative when trianing the flow model using a pointcloud autoencoder

### ISSUE THREAD

**Issue by L-Aidan** [AutodeskAILab/Clip-Forge#15]
I found the logdet becomes 200+ after the first batchnormFlow model, is this related with the input of the flow model(the output of the autoencoder)?

**Comment by L-Aidan** [comment]
I found the loss would be negative even when training the voxel with pretrained ckpt. Does anyone else meet this problem?

**Comment by happysxpp** [comment]
the same problem
